src/Hocon.Configuration/Config.cs

```diff
@@ -42,6 +42,7 @@ protected Config()
         protected Config(HoconValue value)
         {
             Value = (HoconValue)value.Clone(null);
+            Root = (HoconValue)value.Clone(null);
```

Comment:
Does `Root == Value` ?

Reply:
No, they're different instances, the original value is cloned twice, one to the `Value` property, the second is into the `Root` property.

Reply:
but at this point of the code, they are `Equal`, but not `ReferenceEqual`